src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,44 @@
+package duke.task;
+
+/**
+ * Abstract class representing a task item.
+ * A task contains a description as well as a flag of whether it has been completed.
+ */
+public abstract class Task {
+    protected final String description;
+    protected boolean isDone;
```

Comment:
This access modifier can be set to private, since there's a setter method and a getter method for the correct symbols already implemented.

Reply:
Good catch!